pkg/kafka/config.go

```diff
@@ -54,6 +45,9 @@ type Config struct {
 	DialTimeout  time.Duration `yaml:"dial_timeout"`
 	WriteTimeout time.Duration `yaml:"write_timeout"`
 
+	SASLUsername string         `yaml:"sasl_username"`
```

Comment:
Shouldn't you regen the doc ?

Reply:
Yes, I have a pre-commit check set up which normally does it for me on `git push` but I had to skip it this time and rely on CI!